cmd/vsphere-xcopy-volume-populator/internal/populator/secure-vmkfstools-wrapper.py

```diff
@@ -0,0 +1,292 @@
+#!/usr/bin/python
```

Comment:
why is this an almost identical replica of vmkfstools-wrapper/vmkfstools-wrapper.py? we can't merge them?